src/pages/Post/Post.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable prettier/prettier */
+import './Post.css'
+
+import axios from 'axios'
+import React, {useEffect, useState } from 'react'
+
+interface IPost {
+    id: number,
+    links: {id: number, caption: string, url: string}[],
+    caption: string, 
+    short_text: string, 
+    details: string,
+    added_at: string,
+    show_after: string,
+    disable_after: string
+}
+
+const defaultPosts: IPost[] = []
+
+export const Posts: React.FC = () => {
+    const [posts, setPosts]: [IPost[], (posts: IPost[]) => void] = useState(defaultPosts)
+    const [loading, setLoading]: [boolean, (loading: boolean) => void] = React.useState<boolean>(true)
+    const [error, setError]: [string, (error: string) => void] = React.useState("")
+    
+    useEffect(() => {
+        axios.get<IPost[]>('/api/cms/post/visible/', {
+            headers: {
+                "Content-type": "application/json"
+            }
+        })
+        .then(response => {
+            setPosts(response.data)
+            setLoading(false)
+        })
+        .catch(ex => {
+            const error =
+            ex.response.status === 404
+                ? "Resource Not found"
+                : "An unexpected error has occurred"
+            setError(error)
+            setLoading(false)
+        })
+    }, [])
+
+    return(
+        <div id="posts">
+            <ul id="post">
+                {posts.map((post) => (
+                    <li key={post.id}>
+                        <h2 id="bold">{post.caption}</h2>

```

Comment:
toto `id="bold"` nejak defaultne funguje na urcenie bold stylu? lebo tiez som nevidel nikde definiciu

Reply:
to je potom v cssku definovane, ale neviem ci sa to tak ma robit :D